app/static/js/createEvents.js

```diff
@@ -56,7 +56,7 @@ $(document).ready(function() {
   // everything except Chrome
   if (navigator.userAgent.indexOf("Chrome") == -1) {
     $('input.timepicker').timepicker({
-             timeFormat : 'hh:mm p',
+             timeFormat : 'h:mm p',
```

Comment:
This does not appear to have any effect. What was this change intended to do?

Reply:
This change lets you type in only one digit except when you start with 1 because it can be 10,11,12. Basically, I switched from 24-hour format to 12-hour format so that when you press anything more than 1, it will automatically take you to the minutes, so you don't have to press any keys or do anything manually